mis_builder/static/src/components/mis_report_widget.css

```diff
@@ -1,3 +1,22 @@
+.oe_mis_builder_content {
+    max-width: 98vw;
+}
+
+.oe_mis_builder_content .o_list_renderer {
+    width: 100%;
+    overflow: auto;
+    position: relative;
+    height: 95vh; /*limit the height in order for the vertical scroll to work  */
+}
+
+.oe_mis_builder_content .o_list_renderer thead th {
+    position: -webkit-sticky; /* Pour Safari */
+    position: sticky;
+    top: 0;
+    background-color: var(--gray-300);
+    z-index: 1;
+    border: 0.2px solid var(--gray-100);
```

Comment:
In my tests (firefox & chrome on linux), z-index and border don't make much difference. Are they important in other browsers?